torch/testing/_internal/common_utils.py

```diff
@@ -1987,22 +1989,26 @@ def make_tensor(size, device: torch.device, dtype: torch.dtype, *, low=None, hig
         complex type and 1 if integer/boolean type.
     """
 
-    assert low is None or low < 9, "low value too high!"
-    assert high is None or high > -9, "high value too low!"
+    if low is not None and high is not None:
+        assert low <= high, "Expected low value to be lower than or equal to high, but found higher instead."
+        assert high >= low, "Expected high value to be higher than or equal to low, but found lower instead."
```

Comment:
1. These are redundant since both statements test exactly the same thing.
2. For user facing errors we should prefer more expressive error types. Other than for testing functions `assert` should not be relied on during normal operation, since they can be disabled completely by the [`-O` flag](https://docs.python.org/3/using/cmdline.html#cmdoption-o).
3. We could make the error message more expressive by reporting the passed values back to the user.

```suggestion
    if low is not None and high is not None and low > high:
        raise ValueError(f"Expected low <= high, but got {low} > {high}")
```